src/encoded/tests/test_types_utils.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+
+
+@pytest.fixture
+def fastq_no_replicate(award, experiment, lab, platform1):
```

Comment:
this is duplicated from test_file.py right?   Probably their should be a single fastq fixture in a shared place;  datafixtures.py?  Or if it's distinct, it should be a different name.

Reply:
I pulled duplicates into datafixtures. Works fine.